src/third-party/PhishingController.ts

```diff
@@ -238,16 +246,17 @@ export class PhishingController extends BaseController<
     this.update({
       phishing: configs,
     });
-
-    this.lastFetched = Date.now();
   }
 
   private async queryConfig<ResponseType>(
     input: RequestInfo,
   ): Promise<ResponseType | null> {
-    const response = await fetch(input, { cache: 'no-cache' });
+    // We ignore network failures here instead of bubbling them up
+    const response = await safelyExecute(() =>
```

Comment:
It would be better to catch this error in `test`, because that's the place where we want to continue executing if this fails.

If I'm explicitly calling `updatePhishingLists`, I would expect it to throw an error if it fails. But if I'm calling `test` and the update fails, I still want to get the best answer possible about whether the domain is suspected of being malicious or not.

Reply:
It looks like this still hasn't been addressed; the error suppression was moved, but we're still suppressing `updatePhishingLists` errors, which isn't ideal.

As we discussed on Slack though, there is the additional problem of one config failure affecting another, and that's challenging to handle without suppressing `updatePhishingLists` config update errors. So maybe we can clean up error handling in a separate PR and not have that as a blocker here. We already suppress certain types of config update errors today (non-network errors).

Reply:
I created a new issue to improve the error handling: #923